editors/emacs/Portfile

```diff
@@ -43,6 +43,7 @@ configure.args  --disable-silent-rules \
                 --without-gconf \
                 --without-libotf \
                 --without-m17n-flt \
+                --with-libgmp \
```

Comment:
Isn't it uses it by default?

> * Installation Changes in Emacs 27.1
> 
> ** Emacs now uses GMP, the GNU Multiple Precision library.
> By default, if 'configure' does not find a suitable libgmp, it
> arranges for the included mini-gmp library to be built and used.
> The new configure option '--without-libgmp' uses mini-gmp even if a
> suitable libgmp is available.

Reply:
The configure script cannot find macport's gmp unless `--with-libgmp` is given. The default is to use the vendored mini-gmp, not the libgmp installed.

Reply:
@wyuenho this is wired because:
```
√ ~ % otool -L $(which emacs) | grep gmp
	/opt/local/lib/libgmp.10.dylib (compatibility version 15.0.0, current version 15.1.0)
√ ~ % 
```

Reply:
Yes? What's the concern?

```
 otool -L $(which emacs)
/opt/local/bin/emacs:
	/opt/local/lib/libxml2.2.dylib (compatibility version 12.0.0, current version 12.12.0)
	/opt/local/lib/libncurses.6.dylib (compatibility version 6.0.0, current version 6.0.0)
	/opt/local/lib/libgnutls.30.dylib (compatibility version 59.0.0, current version 59.2.0)
	/opt/local/lib/liblcms2.2.dylib (compatibility version 3.0.0, current version 3.12.0)
	/opt/local/lib/libz.1.dylib (compatibility version 1.0.0, current version 1.2.11)
	/opt/local/lib/libjansson.4.dylib (compatibility version 18.0.0, current version 18.0.0)
	/opt/local/lib/libgmp.10.dylib (compatibility version 15.0.0, current version 15.1.0)
	/opt/local/lib/gcc11/libgccjit.0.dylib (compatibility version 0.0.0, current version 0.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1292.100.5)
```

Reply:
You do need the flag for whatever reason.

Reply:
@wyuenho my emacs was build before your changes. As far as I can see your is also linked with `libgmp.10.dylib`.

Reply:
@catap that's the point :) We don't want the default.

Reply:
Have you done a selfupdate and reinstall recently?

Reply:
@wyuenho yes. I'm on `emacs-app-devel @20210917_0+nativecomp+rsvg`